lib/SIL/Verifier/SILVerifier.cpp

```diff
@@ -7201,7 +7201,7 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
     case SILLinkage::Public:
     case SILLinkage::Package:
     case SILLinkage::Shared:
-      require(F->isDefinition() || F->hasForeignBody(),
+      require(F->isDefinition() || F->hasForeignBody() || F->isZombie(),
```

Comment:
Zombie functions should not be verified at all. For most parts of the compiler - including the verifier - they don't exist.

Reply:
Thanks. Let me see why I was hitting this failure.